fcrepo4/fcrepo4
# changed the mapping of fedora ids to ocfl ids to be one-to-one

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3317

# What does this Pull Request do?

I didn't read the ticket (which I had also written...) carefully enough, and I mistakenly updated how Fedora IDs are mapped to OCFL IDs before realizing that I was actually just supposed to be fixing a bug related to not using the mapping index when adding resources to AGs. So... I did them both.

The changes here are that now Fedora IDs are mapped directly to OCFL IDs, and the resource creation code correctly checks the mapping index for the OCFL ID when it adds a resources to an AG.

# How should this be tested?

Start Fedora and create one or more resources. Check the OCFL storage root and you will find objects with OCFL IDs that match their corresponding Fedora ID.

# Interested parties
@fcrepo4/committers
